src/badges.js

```diff
@@ -1153,7 +1152,7 @@ const BADGE_SITES = [
     style: leftMarginStyle
   },
   {
-    name: 'scholar.google.com',
+    name: 'scholar.google',
```

Comment:
does this mean it will no longer work on e.g. `scholar.google.co.uk`?

Reply:
lol I read the diff the wrong way around I think for some reason, it was late in the day here!